xbmc/cores/VideoPlayer/DVDCodecs/Audio/DVDAudioCodecFFmpeg.cpp

```diff
@@ -65,7 +65,7 @@ bool CDVDAudioCodecFFmpeg::Open(CDVDStreamInfo &hints, CDVDCodecOptions &options
       allowdtshddecode = atoi(it->m_value.c_str());
 
   if (hints.codec == AV_CODEC_ID_DTS && allowdtshddecode)
-    pCodec = avcodec_find_decoder_by_name("libdcadec");
```

Comment:
the point there is no option anymore to allow dtdhd or are you aware of any?  explicitly selecting dcadec for DTS is pointless because it is the only and default codec for dts.